lib/algolia/helper.js

```diff
@@ -25,11 +25,11 @@ exports.getTitle = function getTitle(content) {
 exports.getFaqLink = function getFaqLink(platform, slug) {
     let url = `/${platform}/v2`;
     switch (platform) {
-        case 'javascript':
-            url = `${url}/how--to-guides/debugging/faq#${slug}`;
+        case 'server-side':
+            url = `${url}/foundation/faq#${slug}`;

```

Comment:
the URL is https://www.100ms.live/docs/server-side/v2/introduction/faq

Reply:
Right, thanks!